src/SFML/Graphics/Font.cpp

```diff
@@ -503,7 +503,7 @@ Font::Page& Font::loadPage(unsigned int characterSize) const
 
 
 ////////////////////////////////////////////////////////////
-Glyph Font::loadGlyph(std::uint32_t codePoint, unsigned int characterSize, bool bold, float outlineThickness) const
+Glyph Font::loadGlyph(char32_t codePoint, unsigned int characterSize, bool bold, float outlineThickness) const
```

Comment:
Is `char32_t` implicitly convertible to `FT_ULong` which is `unsigned long`?

Reply:
I suppose so? We have all the compiler warnings cranked up so I'm pretty sure we'd get a CI failure if that conversion was potentially lossy.